cloudify_cli/config/parser_config.py

```diff
@@ -548,75 +548,74 @@ def parser_config():
-                            '-s,--snapshot-id': snapshot_id_argument('The id of the snapshot'),
+                            '-s,--snapshot-id': snapshot_id_argument('The ID of the snapshot to restore'),
                             '--without-deployments-envs': {
                                 'dest': 'without_deployments_envs',
                                 'action': 'store_true',
-                                'help': 'Restore snapshot without deployment environments'
+                                'help': 'Restore a snapshot (excluding existing deployments)'
                             },
                             '-f,--force':
                                 force_argument(
                                         hlp='Force restoring the snapshot on '
                                             'a Manager with existing '
-                                            'blueprints and/or deployments')
+                                            'blueprints or deployments')
                         },
-                        'help': 'Restore manager state to a specific snapshot',
+                        'help': "Restore a Manager using a snapshot",
                         'handler': cfy.snapshots.restore
                     }
                 }
             },
             'agents': {
-                'help': "Manage Cloudify's Agents",
+                'help': "Handle a deployment's agents",
                 'sub_commands': {
                     'install': {
                         'arguments': {
                             '-d,--deployment-id': deployment_id_argument(
-                                hlp='The id of the deployment to install '
+                                hlp='The ID of the deployment to install '
                                     'agents for. If omitted, this will '
                                     'install agents for all deployments'
                             ),
                             '-l,--include-logs': include_logs_argument()
                         },
-                        'help':'Install agents on deployments',
+                        'help':'Install agents for existing deployments',
                         'handler': cfy.agents.install
                     }
                 }
             },
             'deployments': {
-                'help': "Manage Cloudify's Deployments",
+                'help': "Handle deployments on the Manager",
                 'sub_commands': {
                     'create': {
                         'arguments': {
                             '-d,--deployment-id': make_required(
                                 remove_completer(
                                     deployment_id_argument(
-                                        hlp='A unique id that will be assigned'
-                                            ' to the created deployment'
+                                        hlp='A unique ID to assign to the created deployment'
```

Comment:
compare with `cfy snapshots create --snapshot-id`: 
`A unique ID for the snapshot`
BTW, isn't an ID unique by definition?